ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -0,0 +1,155 @@
+import ctypes
+import logging
+import time
+from typing import TYPE_CHECKING
+
+from six import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Union
+
+    from ddtrace.appsec.ddwaf.ddwaf_types import DDWafRulesType
+
+
+LOGGER = logging.getLogger(__name__)  # type: logging.Logger
```

Comment:
please prefer the internal logger which adds a rate limiter to ensure we don't accidentally spam a user's logs if we have a warning/error in a hot path.

https://github.com/DataDog/dd-trace-py/blob/903ac658c98c2edf0fd708ad551fadc4665e5317/ddtrace/internal/logger.py